editor/views/rai.py

```diff
@@ -0,0 +1,262 @@
+                on_change=handle_rai_change,
+                args=(RaiEvent.RAI_DATA_ANNOTATION_DEMOGRAPHICS, metadata, key),
+            )
+            key = "metadata-data-annotation-tools"
+            st.text_area(
+                label=(
+                    "**Tools**. List of software used for data annotation ( e.g. concept extraction, NER, and additional characteristics of the tools used for annotation to allow for replication or extension)  "
+                ),
+                key=key,
+                value=metadata.machine_annotation_tools,
+                on_change=handle_rai_change,
+                args=(RaiEvent.RAI_DATA_ANNOTATION_TOOLS, metadata, key),
+            )
+            key = "metadata-data-annotation-per-item"
+            st.text_area(
+                label=(
+                    "**Annotation per item**. Number of human labels per dataset item  "
+                ),
+                key=key,
+                value=metadata.annotation_per_item,
+                on_change=handle_rai_change,
+                args=(RaiEvent.RAI_DATA_ANNOTATION_PER_ITEM, metadata, key),
+            )
+        with st.expander("**Data Preprocessing**", expanded=False):
+            oneToManyProperty(
+                "**Protocols**",
+                metadata,
+                metadata.data_preprocessing_protocol,
+                "metadata-data-preprocessing-protocol_",
+                "Description of data manipulation process if applicable ",
+                RaiEvent.RAI_DATA_PREPROCESSING_PROTOCOL,
+            )
+
+            key = "metadata-data-manipulation-protocol"
+            st.text_area(
+                label=(
+                    "**Manipulation**. Description of data manipulation process if applicable    "
+                ),
+                key=key,
+                value=metadata.data_manipulation_protocol,
+                on_change=handle_rai_change,
+                args=(RaiEvent.RAI_DATA_MANIPULATION_PROTOCOL, metadata, key),
+            )
+            key = "metadata-data-imputation-protocol"
+            st.text_area(
+                label=(
+                    "**Imputation**. Description of data imputation process if applicable  "
+                ),
+                key=key,
+                value=metadata.data_imputation_protocol,
+                on_change=handle_rai_change,
+                args=(RaiEvent.RAI_DATA_IMPUTATION_PROTOCOL, metadata, key),
+            )
+
+    with col2.expander("**Data uses and social impact**", expanded=True):
+        # Data use cases
+        oneToManyProperty(
+            "**Use cases**",
+            metadata,
+            metadata.data_use_cases,
+            "metadata-data-use-cases_",
+            "Dataset use case - training, testing, validation, development or production use, fine tuning, others (please specify), usage guidelines, recommended uses, etc.",
+            RaiEvent.RAI_DATA_USE_CASES,
+        )
+
+        # Data biases
```

Comment:
I would remove all the comments as the title is explicit enough

Reply:
removed